Cabal/src/Distribution/Simple/BuildPaths.hs

```diff
@@ -192,7 +192,8 @@ getSourceFiles verbosity dirs modules = flip traverse modules $ \m ->
     findFileWithExtension ["hs", "lhs", "hsig", "lhsig"] dirs (ModuleName.toFilePath m)
       >>= maybe (notFound m) (return . normalise)
   where
-    notFound module_ = die' verbosity $ "can't find source for module " ++ prettyShow module_
+    notFound module_ =
+      dieWithException verbosity $ CantFindSoureModule module_
```

Comment:
OOC, why is it spelled "Soure" instead of "Source"?

Reply:
It's a typo?

Reply:
```suggestion
      dieWithException verbosity $ CantFindSourceModule module_
```

Reply:
Yes it was a typo, Thanks for pointing out. Changed to "CantFindSourceModule"